### ISSUE TITLE
BCCd responses not matched to right request

### ISSUE THREAD

**Issue by skenaja** [mysociety/alaveteli#32]
Examples of incoming mails that aren't automatically redirected to the right request:

1) 1 WDTK request email address, 1 non-WDTK address.
https://www.whatdotheyknow.com/admin/raw_emails/156083

```
To: Andrew Greenhalgh <request-64044-ae698286@whatdotheyknow.com>, FOI <FOI@sheffield.gov.uk>
```

2) request blind-copied (BCC:) to request.  This example was intended to be BCC'ed to 7 diff't requests...
https://www.whatdotheyknow.com/admin/raw_emails/153576

I'll add more in the comments as they arrive

**Comment by olineham** [comment]
Have had this issue in NZ too - authority emailed "To:" themselves and BCC'd several requests (and probably media - was a high profile story). Therefore there was no request information at all to work with in the headers. We're using postfix. I will look to get X-Envelope-To prepended to make this easier in future (I had to read our postfix log to work out the requests) but would be good to have Alaveteli use this header itself.

**Comment by garethrees** [comment]
I'm going to close this now.

The feedback so far is that we don't get Bcc responses that often, and the additional check in the holding pen seems like a good idea. https://github.com/mysociety/alaveteli/pull/4529 makes it easier to identify multiple Bcc recipients.

We're [explicitly not auto-delivering](https://github.com/mysociety/alaveteli/blob/0.31.0.3/app/mailers/request_mailer.rb#L219-L220) Bcc responses. It wouldn't be too difficult to do so if we wanted to, but it doesn't seem like its a big enough problem to spend time on.

**Comment by garethrees** [comment]
I've just merged #4529 which extracts a bunch of headers so that its easier to spot these:

![screen shot 2018-02-20 at 14 28 27](https://user-images.githubusercontent.com/282788/36429343-61cc297c-164a-11e8-8bd0-916af7c78be8.png)

This will be out on WDTK soon.
